src/providers/boleto-api-bradesco-shopfacil/formatter.js

```diff
@@ -0,0 +1,18 @@
+const moment = require('moment')
+
+const formatDate = timestamp => moment(timestamp).format('YYYY-MM-DD')
+
+const getDocumentType = (documentNumber) => {
+  const documentLength = documentNumber.length
```

Comment:
O seguro morreu de velho 😂 
```suggestion
  const documentLength = String(documentNumber).trim().length
```